designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -1264,7 +1277,9 @@ private fun SquooshChildLayout(modifier: Modifier, child: SquooshChildComposable
             // Place all children at the origin (with the same size constraints that
             // we were given).
             layout(layoutWidth, layoutHeight) {
-                placeables.forEach { child -> child.placeRelative(0, 0) }
+                val x = -scrollOffset.value.x.roundToInt()
```

Comment:
I think reading `scrollOffset` here means that layout will get re-run whenever that state value changes. So I suspect there's a performance impact here, but I'm not sure if there's an alternative in Compose (like rendering the offset only until there's an event, and then doing the layout).

Or maybe Compose layout is fast enough that this is a non-issue...